README.md

```diff
@@ -44,23 +44,26 @@ A tutorial on MO-Gymnasium and MORL-Baselines is also available: [![Open in Cola
+| [MPMOQLearning](https://github.com/LucasAlegre/morl-baselines/blob/main/morl_baselines/multi_policy/multi_policy_moqlearning/mp_mo_q_learning.py)  (outer loop MOQL)          | Multi               | SER                         | Discrete              | Discrete              | [Paper](https://www.researchgate.net/publication/235698665_Scalarized_Multi-Objective_Reinforcement_Learning_Novel_Design_Techniques)                    |
+| [Optimistic Linear Support (OLS)](https://github.com/LucasAlegre/morl-baselines/blob/main/morl_baselines/multi_policy/ols/ols.py)                                             | Multi               | SER                         | /                     | /                     | Section 3.3 of the [thesis](http://roijers.info/pub/thesis.pdf)                                                                                          |
+| [Expected Utility Policy Gradient (EUPG)](https://github.com/LucasAlegre/morl-baselines/blob/main/morl_baselines/single_policy/esr/eupg.py)                                   | Single              | ESR                         | Discrete              | Discrete              | [Paper](https://www.researchgate.net/publication/328718263_Multi-objective_Reinforcement_Learning_for_the_Expected_Utility_of_the_Return)                |
+| [Iterated Pareto Referent Optimisation (IPRO)](https://github.com/LucasAlegre/morl-baselines/blob/main/morl_baselines/multi_policy/ipro/ipro.py)                              | Multi               | SER                         | Continuous            | Discrete              | [Paper](https://arxiv.org/abs/2402.07182)                                                                                                                |
+| [Iterated Pareto Referent Optimisation-2D (IPRO-2D)](https://github.com/LucasAlegre/morl-baselines/blob/main/morl_baselines/multi_policy/ipro/ipro_2d.py) <sup>[2](#f1)</sup> | Multi               | SER                         | Continuous            | Discrete              | [Paper](https://arxiv.org/abs/2402.07182)                                                                                                                |
+| [Non-Linear Multi-Objective PPO (NLMOPPO)](https://github.com/LucasAlegre/morl-baselines/blob/main/morl_baselines/single_policy/ser/nl_mo_ppo.py)                             | Single              | SER                         | Continuous            | Discrete              | Derivation of the policy-gradient update in this [paper](https://cris.vub.be/ws/portalfiles/portal/106449930/96440748.pdf)                               |
 :warning: Some of the algorithms have limited features.
 
-<b id="f2">1</b>: PCN assumes environments with deterministic transitions.
+<b id="f2">1</b>: PCN assumes environments with deterministic transitions. <br>
```

Comment:
Is it possible that the id is messed up, i.e., should be f1 and f2 for the IPRO2D note?

Reply:
Correct, fixed now!